sapper/static/global.css

```diff
@@ -63,34 +63,44 @@ body {
   font-size: 1rem;
 }
 
+.post div,
```

Comment:
We have some styling on this div in tailwind so maybe extract that styling or move it into here. I would vote for doing this styling in tailwind :)

Reply:
Just so we just handle the html blob in plane css

Reply:
Oh I see! Agreed!